docs/sources/setup-grafana/installation/debian/index.md

```diff
@@ -38,112 +28,136 @@ If you install from the APT repository, then Grafana is automatically updated ev
 
 > **Note:** Grafana Enterprise is the recommended and default edition. It is available for free and includes all the features of the OSS edition. You can also upgrade to the [full Enterprise feature set](https://grafana.com/products/enterprise/?utm_source=grafana-install-page), which has support for [Enterprise plugins](https://grafana.com/grafana/plugins/?enterprise=1&utcm_source=grafana-install-page).
 
-#### To install the latest release:
+### Before you begin
```

Comment:
@DanCech , this section is meant to provide a list of prerequisites, for example, the user should have certain permissions in place before they start the task steps. Can you think of any such prereqs? If we don't have any, we can remove this section.

Reply:
will removing